lib/sdotypes.c

```diff
@@ -1552,16 +1574,22 @@ int sdo_hash_read(sdor_t *sdor, sdo_hash_t *hp)
  * @param hp - pointer to the struct of type hash
  * @return none
```

Comment:
Return value is updated from none to bool.

Reply:
Done.